foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/helper/WriteLockManager.java

```diff
@@ -66,7 +66,7 @@
 
     public static final int MAX_WAIT = 600000; //10 mins
     private static final int SIXTY_SECONDS = 60; //1 minute
```

Comment:
Constant should be named for their purpose, not their value.
SIXTY_SECONDS should be SECONDS_BEFORE_HONOUR_INTERRUPT
SEVENTY_SECONDS should be SECONDS_IN_BAD_STATE